packages/@lwc/shared/src/signals.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2024, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+import { isFalse } from './assert';
+
+let trustedSignals: WeakSet<object>;
+
+export function setSignalValidator(signals: WeakSet<object>) {
```

Comment:
```suggestion
export function setSignalTracker(signals: WeakSet<object>) {
```
nit: a set just holds values, it doesn't itself validate anything. 🤷

Reply:
And then maybe also change all the `trusted` to `tracked`? 🤔

Reply:
I suggest `setTrustedSignalSet`